src/prog/thermo.f90

```diff
@@ -164,8 +174,60 @@ subroutine preigf(unit, freq, scale)
 end subroutine preigf
 
 
+! Remove the frequency shift caused by the bias RMSD employed in the SPH calculation:
+subroutine remove_sphshift(env,argParser,mol, hessian, freq)
+   use xtb_hessian
+    real(wp), allocatable, intent(inout) :: freq(:), hessian(:, :)
```

Comment:
Is the `allocatable` statement at this point actually required when allocation happened already before?